pdr_backend/pdr_dashboard/test/resources.py

```diff
@@ -78,3 +84,54 @@ def start_server_and_wait(dash_duo, app):
     dash_duo.wait_for_element("#predictoors_table")
     dash_duo.wait_for_element("#feeds_table tbody tr")
     dash_duo.wait_for_element("#predictoors_table tbody tr")
+
+
+def _navigate_to_feeds_page(dash_duo):
+    dash_duo.wait_for_element("#feeds")
+    dash_duo.find_element("#feeds").click()
+    time.sleep(1)
+
+
+def _remove_tags(text):
```

Comment:
to clarify this functionality, could you add a comment or a docstring giving an example. What tags is it removing and why?